saleor/webhook/payloads.py

```diff
@@ -656,6 +656,21 @@ def serialize_product_channel_listing_payload(channel_listings):
     return channel_listing_payload
 
 
+def serialize_refund_data(refund_data):
+    if order_lines_to_refund := refund_data.get("order_lines_to_refund"):
+        for line_data in order_lines_to_refund:
+            line = line_data["line"]
+            line_data["line"] = str(line.pk)
+            line_data["variant"] = line.variant.pk
+        refund_data["order_lines_to_refund"] = order_lines_to_refund
+    if fulfillment_lines_to_refund := refund_data.get("fulfillment_lines_to_refund"):
+        for line_data in fulfillment_lines_to_refund:
+            line = line_data["line"]
```

Comment:
This could be just one line:
```suggestion
            line_data["line"] = line_data["line"].pk
```